### PR TITLE
Correcting data type in metadata salt condition

### PR DESCRIPTION
**Pull request by tommycz1**
Collecting grains in state salt.minion fail here due to compare char with int.

### CODE REVIEW

File: keystone/meta/salt.yml [hunk 1]
```diff
@@ -30,10 +30,10 @@ minion:
     {%- for profile_name, identity in client.get('server', {}).items() %}
       {%- set protocol = identity.admin.get('protocol', 'http') %}
 
-      {%- if identity.admin.get('api_version', '2') == '3' %}
+      {%- if identity.admin.get('api_version', '2') == 3 %}
```
Comment (cznewt) [inline keystone/meta/salt.yml]: this depends on what you store in metadata, i prefer versions as strings, what do you think?

Comment (cznewt) [reply keystone/meta/salt.yml]: or cast the identity.admin.get('api_version', '2') to string

Comment (tommycz1) [reply keystone/meta/salt.yml]: I tried to define in pillar: api_version: '3' , but this value is still handled like int. Working option is suggested cast to string in jinja condition - this works.

**Comment by tommycz1** [comment]
Hi cznewt, I have changed conditions to cast to string as you proposed. Is this correct?

**Pull request merged**